doc/async_worker_variants.md

```diff
@@ -453,4 +519,28 @@ asynchronous task ends and other data needed for the computation. Once created,
 the only other action needed is to call the `Napi::AsyncProgressQueueWorker::Queue`
 method that will queue the created worker for execution.
 
+Lastly, the following Javascript (ES6+) code would be associated the above example:
+
+```js
+nativeAddon = require('binding.node');
```

Comment:
I think you missed `const`. 
` const { echo } = require('bindings')('addon')`

Reply:
Whoops, you are correct I did...although the `('addon')` part is not needed unless I am crazy here.

Reply:
Whoops nevermind. It's been too many years since I had to do that part itself haha. I actually forgot to copy the last part though, so I will add that too.